packages/realm/src/tests/milestone-4.test.ts

```diff
@@ -18,7 +18,8 @@
 
 import { expect } from "chai";
 
-import { Realm } from "../index";
+import { Realm, Results } from "../index";
```

Comment:
I've noticed that the tests usually import from individual files rather than from `internal.ts`. Should we start using `internal.ts` for the tests as well or does it not matter much in this case?

Reply:
That would probably be good, but then again, I expect to delete these or partly migrate them to integration tests soon.